NAMESPACE

```diff
@@ -25,6 +25,7 @@ export(RunDEAVisuals)
 export(RunDEA_LASSO_consensus)
 export(RunHeatmap)
 export(RunRF)
+export(create_SE)
```

Comment:
this name doesn't seem to be consistent with the other functions, they seem all to start with an uppercase character and roughly follow` camelCase` or at least `CamelCase`

Reply:
fixed